content/en/smart-contracts/fundamentals/erc-20-quickstart/index.md

```diff
@@ -213,6 +213,34 @@ Now that we've successfully compiled our contract, we need to deploy it somewher
 
 On the Filecoin network, a new set of blocks, also called a tipset, is created every thirty seconds. When deploying a contract, the transaction needs to be received by the network, and then the network needs to confirm the contract. This process takes around one to two tipsets to process -- or around 60 to 90 seconds.
 
+## Verify your contract (main net deployment only)
+
+If you are deploying your contract to main net, we highly recommend verifying your contract, to upkeep security and transparency on the network. 
+
+1. In Remix, within the **File Explorer** sidebar tab, under **contracts**, right click on your contract and choose "generate UML" to flatten the .sol file to ensure all components and tasks are included. 
+
+    <img width="779" alt="Screen Shot 2023-04-21 at 4 56 01 PM" src="https://user-images.githubusercontent.com/113331491/233750337-b15e51b3-aad5-4a50-bf62-6fc6625799d2.png">
+
+2. A contract ending in "_ flattened.sol" will be generated below your original contract. Ensure that the license and solidity version in the flattened contract is the same as in your original contract, edit as needed and click "save". 
```

Comment:
```suggestion
2. A contract ending in `_flattened.sol` will be generated below your original contract. Ensure that the license and Solidity version in the flattened contract is the same as in your original contract, edit as needed and click "save". 
```

Reply:
@trruckerfling what do you mean by "edit as needed"?

Reply:
'edit as needed' referred to ensuring you paste the license and solidity version into the flattened contract. could remove the phrase and it wouldnt change the sentence, esp if it is confusing

Reply:
@trruckerfling check out the suggestion -- the filename continues to be incorrect.